web/src/components/NormsList.vue

```diff
@@ -0,0 +1,23 @@
+<script lang="ts" setup>
+import Norms from "../domain/Norm"
```

Comment:
The type is actually called `Norm` in singular form.
I would recommend to remove the default export. So please make `export default Norm` without the `default`. That enforces the usage of the correct name everywhere. No need for a default export anyway. 😊